gradescope-to-db/migrations/20230811013132_regrade_completion.sql

```diff
@@ -0,0 +1,3 @@
+-- Add migration script here
+ALTER TABLE regrade
+ADD completed TINYINT NOT NULL;
```

Comment:
SQLite wants a default value, since existing rows would otherwise default to `NULL`, which is prohibited.
```suggestion
ADD completed TINYINT NOT NULL DEFAULT false;
```